### ISSUE TITLE
Just updated to 2.1.7 but I can't find anything about this release in the repo.

### ISSUE THREAD

**Issue by ablakey** [microsoft/vscode-eslint#1011]
Just got a quasi-suspicious "user needs confirmation before executing code in local library" pop-up. So I went to see what the latest version's code is to verify it's legit.  I cannot find anything about 2.1.7 in this repo. Can someone point me in the right direction?

![image](https://user-images.githubusercontent.com/9021944/87459736-c9a79680-c5d9-11ea-9660-b1a4bcdb77f6.png)

From: https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint
![image](https://user-images.githubusercontent.com/9021944/87460560-0cb63980-c5db-11ea-9027-80ee49828844.png)

@dbaeumer

**Comment by dbaeumer** [comment]
Will be merged in soon.

**Comment by guanzo** [comment]
What if I have multiple projects in the workspace that all have their own eslint installations?

```
projects/
  projectA/ # eslint installed here
  projectB/ # eslint installed here
```

The prompts only ask to use the global eslint installation, or asks to use one of projectA/projectB's eslint for the whole workspace. It doesn't ask to use A's eslint to lint A, and B's eslint to lint B. is this possible?

**Comment by ablakey** [comment]
> Will be merged in soon.

Appreciate it.  I use vscode-eslint constantly and it makes my job easier. Thank you and all the other contributors for the hard work they do.

I was really hoping to see a release tag or PR that I could diff to understand the changes and more easily scrutinize security. It's unfortunate that such a popular extension seems to get updated behind the scenes.

**Comment by tjohnsonBCM** [comment]
Please add an option to disable the prompts. Adding them is a significant change that I would have not expected on a patch release. And there is no mention in the release notes of an option allowing us to disable the prompts to restore the old behavior.

![image](https://user-images.githubusercontent.com/50887074/87543762-420e6600-c673-11ea-9f21-9314b1f94513.png)

**Comment by dbaeumer** [comment]
@tjohnsonBCM please see the discussions here on why that got added: https://github.com/microsoft/vscode-eslint/issues/1012#issuecomment-659097449

I have merged the changes into the release/2.1.x branch in this repository.
